### ISSUE TITLE
'vike/client/router' does not provide an export named 'prefetch'

### ISSUE THREAD

**Issue by dajinchu** [vikejs/vike#1471]
### Description

Reproduction:
1. use prefetch anywhere (I put it in `examples/react-full/index/Page.tsx`)
2. run `npm run preview`

Works fine in dev mode.

```
file:///Users/dajinchu/Alignable/vike/examples/react-full/dist/server/entries/pages_index.mjs:3
import { prefetch, navigate } from "vike/client/router";
         ^^^^^^^^
SyntaxError: The requested module 'vike/client/router' does not provide an export named 'prefetch'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
    at ModuleJob.run (node:internal/modules/esm/module_job:191:5)
    at ModuleLoader.import (node:internal/modules/esm/loader:336:24)
    at loadConfigValues (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vike/dist/esm/shared/page-configs/loadConfigValues.js:11:32)
    at file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vike/dist/esm/node/prerender/runPrerender.js:177:34
Error: [vike@0.4.160][Bug] You stumbled upon a Vike bug. Go to https://github.com/vikejs/vike/issues/new and copy-paste this error. A maintainer will fix the bug (usually under 24 hours).
    at extractFromPath (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vike/dist/esm/node/runtime/renderPage/logHintForCjsEsmError.js:228:5)
    at parseImportFrom (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vike/dist/esm/node/runtime/renderPage/logHintForCjsEsmError.js:173:25)
    at isCjsEsmError (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vike/dist/esm/node/runtime/renderPage/logHintForCjsEsmError.js:117:30)
    at getHint (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vike/dist/esm/node/runtime/renderPage/logHintForCjsEsmError.js:21:17)
    at logHintForCjsEsmError (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vike/dist/esm/node/runtime/renderPage/logHintForCjsEsmError.js:12:18)
    at runPrerenderFromAutoFullBuild (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vike/dist/esm/node/prerender/runPrerender.js:51:9)
    at triggerFullBuild (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vike/dist/esm/node/plugin/plugins/autoFullBuild.js:89:9)
    at Object.handler (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vike/dist/esm/node/plugin/plugins/autoFullBuild.js:30:25)
    at async Promise.all (index 0)
    at PluginDriver.hookParallel (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/rollup/dist/es/shared/node-entry.js:18620:9)
    at file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/rollup/dist/es/shared/node-entry.js:19612:13
    at catchUnfinishedHookActions (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/rollup/dist/es/shared/node-entry.js:19036:16)
    at build (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vite/dist/node/chunks/dep-9A4-l-43.js:67181:22)
    at CAC.<anonymous> (file:///Users/dajinchu/Alignable/vike/examples/react-full/node_modules/vite/dist/node/cli.js:845:9)
```

**Comment by brillout** [comment]
I could reproduce indeed, let me fix this.

**Comment by brillout** [comment]
Fix pre-released in `0.4.160-commit-937d8d5`. (The second issue `vike@0.4.160][Bug] You stumbled upon a Vike bug.` was already fixed on `main`.)

Thank you for opening the issue, let me know if there is anything else. (And thank you for sponsoring, it's making a huge difference :green_heart:)
